BookingsApi/BookingsApi.Infrastructure.Services/Dtos/HearingAllocationDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace BookingsApi.Infrastructure.Services.Dtos
+{
+    public class HearingAllocationDto : HearingDto 
```

Comment:
The judge display name seems to only be needed for the toast notification. This value can be pulled in from the cache like the other event handlers, thus simplifying the event

Reply:
I am sorry but I don't agree with that. Booking api is making the fetching and passing to BS only the needed text to be passed then to VideoWeb